homework-2/src/scss/core/_common.scss

```diff
@@ -0,0 +1,61 @@
+*,
+*:after,
+*:before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a,
+button {
+  background-color: transparent;
+  border: none;
+  outline: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+button{
+  box-shadow: 0px 6px 12px rgba(127, 127, 127, 0.3);
+}
+
+a,
+p,
+h1,
+li{
+  font-weight: 400;
+}
+
+a,
+p,
+li{
+  // font-family: 'Aclonica', sans-serif;
+  font-family: $font-text;
+}
+
+h1{
+  // font-family: 'Actor', sans-serif;
+  font-family: $font-sub-title;
+}
+
+hr{
```

Comment:
don't use `hr` use css styles instead for such purpose

Reply:
ok. done